implementations/rust/ockam/ockam_command/src/enroll.rs

```diff
@@ -51,8 +51,9 @@ async fn run_impl(ctx: &Context, opts: CommandGlobalOpts, cmd: EnrollCommand) ->
 
     let cloud_opts = cmd.cloud_opts.clone();
     let space = default_space(ctx, &opts, &cloud_opts, &node_name).await?;
-    default_project(ctx, &opts, &cloud_opts, &node_name, &space).await?;
+    let project = default_project(ctx, &opts, &cloud_opts, &node_name, &space).await?;
     update_enrolled_identity(ctx, &opts, &node_name).await?;
+    project_enroll_admin(ctx, &opts, &node_name, &project).await?;
```

Comment:
this must be done for all projects he has, not just one.

Reply:
added here: https://github.com/build-trust/ockam/pull/4152/commits/cc7a4a9eae031a76882fab510ca523d92882e844